### DISCUSSION TITLE
Will this work with Tor Browser?

### DISCUSSION THREAD

**Discussion by Aleksandra0056** [black7375/Firefox-UI-Fix#571] [Q&A] [answered]
And if I install this in Tor Browser, will it affect my privacy?

**Comment by JohnyP36** [comment]
I don't think this will affect your privacy as it does not sent info to a server.

**Comment by black7375** [comment, marked as answer]
Tor browser is supported.
There is only CSS, so it should not be theoretically affected.
If there is a problem, please report it.

**Reply by Aleksandra0056** [reply]
Not Thor, but Tor: https://torproject.org/

**Reply by JohnyP36** [reply]
> There is only CSS

So, it should not affect your privacy.
